Microsoft.Toolkit.Services/Microsoft.Toolkit.Services.csproj

```diff
@@ -19,6 +19,11 @@
     <DefineConstants Condition="'$(DisableImplicitFrameworkDefines)' != 'true'">$(DefineConstants);WINRT</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net462'">
```

Comment:
Woah, hadn't noticed the toolkit was still supporting .NET Framework 4.6.2 too! 😄